### ISSUE TITLE
Bleak version not recognised

### ISSUE THREAD

**Issue by macray76** [SanaaHamel/nevermore-controller#12]
Running the ./install-klipper-module.bash I get the following error:
```
ERROR: Ignored the following versions that require a different python version: 0.21.0 Requires-Python >=3.8,<3.13; 0.21.1 Requires-Python >=3.8,<3.13
ERROR: Could not find a version that satisfies the requirement bleak>=0.21.1<1 (from versions: 0.2.0, 0.2.2, 0.2.3, 0.2.4, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.5.1, 0.6.0, 0.6.1, 0.6.2, 0.6.4, 0.7.0, 0.7.1, 0.8.0, 0.9.0, 0.9.1, 0.10.0, 0.11.0, 0.12.0, 0.12.1, 0.13.0, 0.14.0, 0.14.1, 0.14.2, 0.14.3, 0.15.0, 0.15.1, 0.16.0, 0.17.0, 0.18.0, 0.18.1, 0.19.0, 0.19.1, 0.19.2, 0.19.3, 0.19.4, 0.19.5, 0.20.0, 0.20.1, 0.20.2)
ERROR: No matching distribution found for bleak>=0.21.1<1
```

Current python version is 3.12.3
Installed bleak version 0.21.1 as well as rolling back to 0.20.2 but the error still remains.

**Comment by macray76** [comment]
I changed the requirements.txt to 0.20.2 and then ran the installation and it installed the controller. However, I know this isn't a fix because I can't get klipper to connect to the bluetooth (despite it being recognised when running bluetoothctl).

I originally had python version 3.7 before upgrading. I did try setting the python symlink back to version 3.7 and then reinstalling but the same issue occured.
